src/renderer/containers/QuickStart/DolphinMigrationStep.tsx

```diff
@@ -0,0 +1,152 @@
+/** @jsx jsx */
+import { ipc_copyDolphin } from "@dolphin/ipc";
+import { DolphinLaunchType } from "@dolphin/types";
+import { css, jsx } from "@emotion/react";
+import styled from "@emotion/styled";
+import Box from "@material-ui/core/Box";
+import Container from "@material-ui/core/Container";
+import FormHelperText from "@material-ui/core/FormHelperText";
+import * as fs from "fs-extra";
+import path from "path";
+import React from "react";
+import { Controller, useForm } from "react-hook-form";
+
+import { Button } from "@/components/FormInputs/Button";
+import { PathInput } from "@/components/PathInput";
+import { useDesktopApp } from "@/lib/hooks/useQuickStart";
+
+import { QuickStartHeader } from "./QuickStartHeader";
+
+const MIGRATE_TEXT = "Migrate Dolphin";
+const SKIP_TEXT = "Skip Migration";
+
+export const MigrateDolphinStep: React.FC = () => {
+  const [netplayMigration, setNetplayMigration] = React.useState(true);
+  const oldDesktopAppPath = useDesktopApp((store) => store.path);
+  const setExists = useDesktopApp((store) => store.setExists);
+  const oldDesktopDolphin = path.join(oldDesktopAppPath, "dolphin");
+
+  const migrateNetplayDolphin = async () => {
+    await ipc_copyDolphin.renderer!.trigger({ dolphinType: DolphinLaunchType.NETPLAY, userPath: netplayPath });
+    setNetplayMigration(false);
+  };
+
+  const deleteOldDesktopAppFolder = async () => {
+    await fs.remove(oldDesktopAppPath);
+    setExists(false);
+  };
```

Comment:
Again this clean up stuff should be handled in the main process side, since websites (the renderer) shouldn't know anything about the filesystem.